client-linuxapp/src/main.rs

```diff
@@ -1218,5 +1226,12 @@ async fn main() -> Result<()> {
         }
     }
     log::info!("Secure Device Onboarding DONE");
+    log::info!("Reboot required? {}", reboot_si_required);
```

Comment:
what's the _si_ here, and elsewhere, is that meant to be is or is it Spanish for yes, or is it a typo, or something else entirely?

Reply:
probably stands for "service info"?

Reply:
yep, Service Info